UIKitMovie/UIKitMovie/Helpers/FileManagerService/FileManagerService.swift

```diff
@@ -0,0 +1,60 @@
+// FileManagerService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Сервис файл менеджер
+final class FileManagerService: FileManagerServiceProtocol {
```

Comment:
```suggestion
final class FileManagerService: FileManagerServiceProtocol {
```
или менеджер или сервис сразу 2 не бывает)